Packs/FidelisEndpoint/Integrations/FidelisEndpoint/FidelisEndpoint.py

```diff
@@ -0,0 +1,1577 @@
+
+    return human_readable, entry_context, scripts
+
+
+def script_manifest_command(client: Client, args: dict):
+    script_id = args.get('script_id')
+    headers = ['Name', 'ID', 'Description', 'Platform', 'Command', 'Questions', 'Priority', 'TimeoutSeconds',
+               'ResultColumns', 'ImpersonationUser', 'ImpersonationPassword', 'WizardOverridePassword']
+    response = client.script_manifest(script_id)
+    data = response.get('data')
+
+    platforms = [k for k, v in data.get('platforms').items() if v]
+
+    contents = {
+        'ID': data.get('id'),
+        'Name': data.get('name'),
+        'Platform': platforms,
+        'Description': data.get('description'),
+        'Priority': data.get('priority'),
+        'ResultColumns': data.get('resultColumns'),
+        'TimeoutSeconds': data.get('timeoutSeconds'),
+        'ImpersonationUser': data.get('impersonationUser'),
+        'ImpersonationPassword': data.get('impersonationPassword'),
+        'Command': data.get('command'),
+        'WizardOverridePassword': data.get('wizardOverridePassword'),
+        'Questions': data.get('questions')
+    }
+
+    entry_context = {'FidelisEndpoint.Script(val.ID && val.ID === obj.ID)': contents}
+    human_readable = tableToMarkdown('Fidelis Endpoint script manifest', contents, headers, removeNull=True)
+
+    return human_readable, entry_context, response
+
+
+def execute_script_command(client: Client, args: dict):
+    script_id = args.get('script_id')
+    time_out = args.get('time_out')
+    endpoint_ip = argToList(args.get('endpoint_ip'))
+    endpoints = client.convert_ip_to_endpoint_id(endpoint_ip)
+    endpoint_id = endpoints.get('data')
+    answer = args.get('answer')
+
+    response = client.execute_script(script_id, endpoint_id, answer, time_out)
+    context = {
+        'ID': script_id,
+        'JobID': response.get('data')
+    }
+    entry_context = {'FidelisEndpoint.Script(val.ID && val.ID === obj.ID)': context}
+
+    return 'The job has been executed successfully', entry_context, response
+
+
+def list_process_command(client: Client, args: dict):
+    endpoint_ip = argToList(args.get('endpoint_ip'))
+    endpoints = client.convert_ip_to_endpoint_id(endpoint_ip)
+    endpoint_id = endpoints.get('data')
+    time_out = args.get('time_out')
+    os = args.get('os')
+    script_id = ''
+
+    if os == 'Windows':
+        script_id = '2d32a530-0716-4542-afdc-8da3bd47d8bf'  # disable-secrets-detection
+
+    if os == 'Linux':
+        script_id = '5e58a0e9-450d-4394-8360-159d5e38c280'  # disable-secrets-detection
+
+    if os == 'macOS':
+        script_id = '020114c2-d000-4876-91b0-97f41a83b067'  # disable-secrets-detection
+
+    response = client.list_process(script_id, time_out, endpoint_id)
+    context = {
+        'ID': script_id,
+        'JobID': response.get('data')
+    }
+    entry_context = {'FidelisEndpoint.Script(val.ID && val.ID === obj.ID)': context}
```

Comment:
the outputs here do not seem relevant to this command

Reply:
This is the only output that returns (the data) the ID is just the unique key to add to the right context path.
The data is basically the same as Job ID. (its a workaround from Fidelis)

Reply:
yea but listing processes, not scripts, so why are we populating these under `Script` ?

Reply:
Will check it again with design. Basically all this commands are script executions.

Reply:
Changed.